go.mod

```diff
@@ -75,7 +76,7 @@ require (
 	github.com/textileio/go-assets v0.0.0-20200430191519-b341e634e2b7
 	github.com/textileio/go-ds-mongo v0.1.5-0.20201230201018-2b7fdca787a5
 	github.com/textileio/go-threads v1.0.3-0.20201216032729-f7b034a0de80
-	github.com/textileio/powergate/v2 v2.2.0
+	github.com/textileio/powergate/v2 v2.2.1-0.20210308170300-1912f81755cf
```

Comment:
Whenever I merge and tag https://github.com/textileio/powergate/pull/795, I'll update to the final tag here.
I'll test things in staging before merging and tagging, then I can wrap up all things.